storage/database/dynamodb.go

```diff
@@ -559,6 +559,12 @@ func (batch *dynamoBatch) Put(key, val []byte) error {
 	return nil
 }
 
+// Delete inserts the a key removal into the batch for later committing.
+func (batch *dynamoBatch) Delete(key []byte) error {
+	logger.CritWithStack("Delete should not be called when using dynamodb batch")
```

Comment:
@ehnuje Currently, I don't know when `Delete` and `Replay` methods of batches are called. Is it okay that `dynamoBatch` raise a panic?

Reply:
@aidan-kwon It is used for Snapshot and will be a problem if we use snapshot on DynamoDB. But for now, it won't be a problem since it is a newly introduced function and not used anywhere.

Reply:
Okay. Let's consider it later when Snapshot is introduced